### ISSUE TITLE
Support for jupyter notebook version 6

### ISSUE THREAD

**Issue by Lucaszw** [danielfrg/s3contents#72]
This package seems to break for jupyter notebook versions >= 6

due to the following line in ipycompact.py
```python
if notebook.version_info[0] >= 6:  # noqa
    raise ImportError("Jupyter Notebook versions 6 and up are not supported.")
```

What is the reason for this?

Thanks!
Lucas

**Comment by rsmulktis** [comment]
Second this. I'm having to force my Jupyter down below 6, which isn't a big deal but will as the project progresses farther away from 5.x.x.

**Comment by chonanlee** [comment]
I'm having this issue too, what's the reason for this?

**Comment by danielfrg** [comment]
There is no reason other than it was not being tested. I removed the hard stop at https://github.com/danielfrg/s3contents/pull/73 and trying to test it now. If it works it will be released today :)

**Comment by danielfrg** [comment]
I just released `0.3.0` to support Notebooks version 6.0.
Note that this doesn't work on Python 2 as I think it's not supported any more for on that jupyter notebook version.

Link: https://pypi.org/project/s3contents/0.3.0/
